gh-repo-stats

```diff
@@ -221,7 +221,7 @@ Header()
     GITHUB_URL="https://api.github.com"
     GRAPHQL_URL="https://api.github.com/graphql"
   else
-    GITHUB_URL+="${GHE_URL}/api/v3"
+    GITHUB_URL="${GHE_URL}/api/v3"
```

Comment:
Seems to be the case. @admiralAwkbar why we appending?

Reply:
if the repo is on GHES the URL is not api.github but ur./api/v3